addons/main/XEH_preInit.sqf

```diff
@@ -11,4 +11,29 @@ GVAR(shareHandlers) = [];
 
 GVAR(blockSuppressionModelCache) = false call CBA_fnc_createNamespace;
 
+if (isServer) then {
+    GVAR(versionLoadedOnServer) = QUOTE(VERSION_STR);
+    publicVariable QGVAR(versionLoadedOnServer);
+} else {
+    0 spawn {
+        if (GVAR(versionLoadedOnServer) isEqualTo QUOTE(VERSION_STR)) exitWith {};
+        "LAMBS DANGER VERSION MISMATCH ERROR!!!!" hintC parseText format [
+            "Lambs Danger Version mismatch Error.<br/>Client Version: %1<br/>Server Version: %2",
+            QUOTE(VERSION_STR),
+            GVAR(versionLoadedOnServer)
+        ];
+        private _error = format [
+            "Lambs Danger Version mismatch Error. Client Version: %1 Server Version: %2",
+            QUOTE(VERSION_STR),
+            GVAR(versionLoadedOnServer)
+        ];
+        while {true} do {
```

Comment:
you sure you want to spam that every second?

Reply:
yes